pkg/k8s/pod.go

```diff
@@ -594,7 +595,7 @@ func (c *Client) prepareInitContainers(config ContainerConfig, init bool) []v1.C
 	return []v1.Container{
 		{
 			Name:  config.Name + initContainerNameSuffix,
-			Image: config.Image,
+			Image: initContainerImage,
```

Comment:
[Q] why does it have to be a fixed image?

Reply:
When it is the same image as the main container, the init container can fail.
The init container needs to create directories, move files, and change the owner and permissions of files. As not each application container has those tools, we need to use an image in the init container that has these tools.